### ISSUE TITLE
Project Revamp

### ISSUE THREAD

**Issue by Foggalong** [numixproject/com.numix.icons_circle#55]
Following on from [this thread](https://github.com/numixproject/numix-core/pull/5149#issuecomment-640189723), started this in what seemed like the most sensible place! If this turns into a more substantial conversation we can move it to a teams discussion board if needed :)

**Comment by Foggalong** [comment]
A potential blocker for having adaptive icons is that I'm not sure sure it's possible to properly handle shadows if you're using them (like the bottom bevel on Square for example) which would mean creating a new icon "type" just for Android.

**Comment by maxtron95** [comment]
The current theme [here](https://github.com/numixproject/com.numix.icons_circle/tree/master/Icons) has icons named `rename_me_...` as icon names. Would it be possible to give them better names in the future?

**Comment by paolorotolo** [comment]
Yes, I would start fresh and possibly rewrite in Kotlin. There's also a MainActivity22 :D

**Comment by maxtron95** [comment]
@paolorotolo I was thinking, Maybe instead of a separate Icon Requests app, we could merge the Icon Requests in the icon theme app like how many android icon themes do it.

**Comment by paolorotolo** [comment]
Yes, if we take the candybar way, the icon request form is already there.

**Comment by maxtron95** [comment]
The question is are we going to use the Candybar template or write the app from scratch?

**Comment by paolorotolo** [comment]
As you want guys. We might try _Candybar_ first and build a simple proof of concept. If we see the library is not so flexible for our needs, I don't have problems rewriting all. If you take out the icons and the xml files, the code shouldn't be too much.

**Comment by Foggalong** [comment]
That works for me!

**Comment by alekksander** [comment]
hello! thanks for the interest guys. just curious, if anyone is still looking into this?
